### PR TITLE
[FR] Add `Client/Menu_tips`

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Pisapou** [comment]
Since French tips are different from the English tips, I added this sentence in the introduction:
`Il est à noter que ces astuces sont différentes des anglaises.` which means `It should be noted that these tips are different from the English ones.`

File: wiki/Client/Menu_tips/fr.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+## Voir aussi
+
+- [Client/Interface](/wiki/Client/Interface)
+- [Client/Éditeur de Beatmap](/wiki/Client/Beatmap_editor)
+- [Client/Références des raccourcis claviers](/wiki/Client/Keyboard_shortcuts)
+
+## Notes
+
+[^frame-limiter]: Aujourd'hui appelé "Limite de FPS".
```
Comment (Pisapou) [inline wiki/Client/Menu_tips/fr.md:125]:
```suggestion
```

File: wiki/Client/Menu_tips/fr.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+- Jetez un coup d'œil sous le capot aux compteurs de performance et activez l'enregistrement des performances verbeuses avec `Ctrl` + `F2` !
+
+## Voir aussi
+
+- [Client/Interface](/wiki/Client/Interface)
+- [Client/Éditeur de Beatmap](/wiki/Client/Beatmap_editor)
+- [Client/Références des raccourcis claviers](/wiki/Client/Keyboard_shortcuts)
+
+## Notes
+
```
Comment (Pisapou) [inline wiki/Client/Menu_tips/fr.md:124]:
```suggestion
```

File: wiki/Client/Menu_tips/fr.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+- Basculez les statistiques avancées des trames / fils avec `Ctrl` + `F11` !
+- Jetez un coup d'œil sous le capot aux compteurs de performance et activez l'enregistrement des performances verbeuses avec `Ctrl` + `F2` !
+
+## Voir aussi
+
+- [Client/Interface](/wiki/Client/Interface)
+- [Client/Éditeur de Beatmap](/wiki/Client/Beatmap_editor)
+- [Client/Références des raccourcis claviers](/wiki/Client/Keyboard_shortcuts)
+
+## Notes
```
Comment (Pisapou) [inline wiki/Client/Menu_tips/fr.md:123]:
```suggestion
```

File: wiki/Client/Menu_tips/fr.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+- Découvrez le système de "playlists", qui permet aux utilisateurs de créer leurs propres classements personnalisés et permanents !
+- Basculez les statistiques avancées des trames / fils avec `Ctrl` + `F11` !
+- Jetez un coup d'œil sous le capot aux compteurs de performance et activez l'enregistrement des performances verbeuses avec `Ctrl` + `F2` !
+
+## Voir aussi
+
+- [Client/Interface](/wiki/Client/Interface)
+- [Client/Éditeur de Beatmap](/wiki/Client/Beatmap_editor)
+- [Client/Références des raccourcis claviers](/wiki/Client/Keyboard_shortcuts)
+
```
Comment (Pisapou) [inline wiki/Client/Menu_tips/fr.md:122]:
```suggestion
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Client/Menu_tips/fr.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+- La recherche au sein des replays est disponible en glissant sur la barre de difficulté en bas de l'écran !
+- La prise en charge du multithreading signifie que même avec des "FPS" faibles, votre saisie et vos jugements seront précis !
+- Essayez de faire défiler vers le bas le panneau de sélection des mods pour trouver un tas de nouveaux mods amusants !
+- La majorité des contenus web (profils, classements, etc.) sont disponibles nativement dans le jeu à partir des icônes de la barre d'outils !
+- Obtenez plus de détails, cachez ou supprimez une beatmap en cliquant avec le bouton droit de la souris sur son panneau lors de la sélection de la beatmap !
+- Toutes les opérations de suppression sont temporaires jusqu'à la sortie. Restaurez le contenu accidentellement supprimé à partir des paramètres de maintenance !
+- Découvrez le système de "playlists", qui permet aux utilisateurs de créer leurs propres classements personnalisés et permanents !
+- Basculez les statistiques avancées des trames / fils avec `Ctrl` + `F11` !
+- Jetez un coup d'œil sous le capot aux compteurs de performance et activez l'enregistrement des performances verbeuses avec `Ctrl` + `F2` !
```
Comment (Pisapou) [inline wiki/Client/Menu_tips/fr.md:115]:
```suggestion
- Jetez un coup d'œil aux compteurs de performance et activez l'enregistrement des performances verbeuses avec `Ctrl` + `F2` !
```

File: wiki/Client/Menu_tips/fr.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+<!-- Source: https://github.com/ppy/osu/blob/master/osu.Game/Screens/Menu/Disclaimer.cs -->
+
+Ces astuces apparaissent dans la [version lazer du client](https://github.com/ppy/osu-lazer), au démarrage, avant la séquence d'introduction.
+
+- Vous pouvez appuyer sur `Ctrl` + `T` n'importe où dans le jeu pour basculer la barre d'outils !
+- Vous pouvez appuyer sur `Ctrl` + `O` n'importe où dans le jeu pour accéder aux options !
+- Tous les paramètres sont dynamiques et prennent effet en temps réel. Essayez de mettre en pause et de changer de skin tout en jouant !
+- De nouvelles fonctionnalités sont mises en ligne à chaque mise à jour. Assurez-vous de rester à jour !
+- Si vous trouvez l'UI trop grande ou trop petite, essayez de régler l'échelle de l'interface dans les paramètres !
```
Comment (Ganondorf-1) [inline wiki/Client/Menu_tips/fr.md:104]:
```suggestion
- Si vous trouvez l'interface trop grande ou trop petite, essayez de régler l'échelle de l'interface dans les paramètres !
```

File: wiki/Client/Menu_tips/fr.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+- Essayez d'ajuster le mode `Screen Scaling` pour changer votre zone de jeu ou d'UI, même en plein écran !
```
Comment (Ganondorf-1) [inline wiki/Client/Menu_tips/fr.md:105]:
```suggestion
- Essayez d'ajuster le mode `Screen Scaling` pour changer votre zone de jeu ou d'interface, même en plein écran !
```

File: wiki/Client/Menu_tips/fr.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+- Ce qui était auparavant "[osu!direct](/wiki/osu!supporter#osu!direct)" est disponible pour tous les utilisateurs comme sur le site web. Vous pouvez y accéder partout en utilisant `Ctrl` + `D` !
+- La recherche au sein des replays est disponible en glissant sur la barre de difficulté en bas de l'écran !
```
Comment (Ganondorf-1) [inline wiki/Client/Menu_tips/fr.md:107]:
```suggestion
- Vous pouvez vous rendre à n'importe quel passage d'un replay en faisant défiler la barre de difficulté en bas de l'écran !
```

File: wiki/Client/Menu_tips/fr.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+- La prise en charge du multithreading signifie que même avec des "FPS" faibles, votre saisie et vos jugements seront précis !
```
Comment (Ganondorf-1) [inline wiki/Client/Menu_tips/fr.md:108]:
```suggestion
- La prise en charge du multithreading signifie que même avec peu de FPS, vos clics et vos jugements seront précis !
```

File: wiki/Client/Menu_tips/fr.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+- Essayez de faire défiler vers le bas le panneau de sélection des mods pour trouver un tas de nouveaux mods amusants !
```
Comment (Ganondorf-1) [inline wiki/Client/Menu_tips/fr.md:109]:
```suggestion
- Essayez de faire défiler le menu de sélection des mods vers le bas pour trouver un tas de nouveaux mods amusants !
```

File: wiki/Client/Menu_tips/fr.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+- La majorité des contenus web (profils, classements, etc.) sont disponibles nativement dans le jeu à partir des icônes de la barre d'outils !
+- Obtenez plus de détails, cachez ou supprimez une beatmap en cliquant avec le bouton droit de la souris sur son panneau lors de la sélection de la beatmap !
```
Comment (Ganondorf-1) [inline wiki/Client/Menu_tips/fr.md:111]:
```suggestion
- Obtenez plus de détails, cachez ou supprimez une beatmap en cliquant dessus avec le bouton droit de la souris sur l'écran de sélection de beatmap !
```

File: wiki/Client/Menu_tips/fr.md [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+- Toutes les opérations de suppression sont temporaires jusqu'à la sortie. Restaurez le contenu accidentellement supprimé à partir des paramètres de maintenance !
```
Comment (Ganondorf-1) [inline wiki/Client/Menu_tips/fr.md:112]:
```suggestion
- Toutes les opérations de suppression sont temporaires jusqu'à la fermeture du jeu. Restaurez le contenu accidentellement supprimé à partir des paramètres de maintenance !
```

File: wiki/Client/Menu_tips/fr.md [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+- Découvrez le système de "playlists", qui permet aux utilisateurs de créer leurs propres classements personnalisés et permanents !
+- Basculez les statistiques avancées des trames / fils avec `Ctrl` + `F11` !
```
Comment (Ganondorf-1) [inline wiki/Client/Menu_tips/fr.md:114]:
```suggestion
- Basculez les statistiques avancées des images / threads avec `Ctrl` + `F11` !
```

File: wiki/Client/Menu_tips/fr.md [hunk 14]
```diff
@@ -0,0 +1,121 @@
...
+- Jetez un coup d'œil aux compteurs de performance et activez l'enregistrement des performances verbeuses avec `Ctrl` + `F2` !
```
Comment (Ganondorf-1) [inline wiki/Client/Menu_tips/fr.md:115]:
```suggestion
- Jetez un coup d'œil aux compteurs de performance et activez l'enregistrement détaillé des performances avec `Ctrl` + `F2` !
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Client/Menu_tips/fr.md [hunk 15]
```diff
@@ -0,0 +1,121 @@
...
+
+- Vous pouvez appuyer sur `Ctrl` + `T` n'importe où dans le jeu pour basculer la barre d'outils !
+- Vous pouvez appuyer sur `Ctrl` + `O` n'importe où dans le jeu pour accéder aux options !
+- Tous les paramètres sont dynamiques et prennent effet en temps réel. Essayez de mettre en pause et de changer de skin tout en jouant !
+- De nouvelles fonctionnalités sont mises en ligne à chaque mise à jour. Assurez-vous de rester à jour !
+- Si vous trouvez l'interface trop grande ou trop petite, essayez de régler l'échelle de l'interface dans les paramètres !
+- Essayez d'ajuster le mode `Screen Scaling` pour changer votre zone de jeu ou d'interface, même en plein écran !
+- Ce qui était auparavant "[osu!direct](/wiki/osu!supporter#osu!direct)" est disponible pour tous les utilisateurs comme sur le site web. Vous pouvez y accéder partout en utilisant `Ctrl` + `D` !
+- Vous pouvez vous rendre à n'importe quel passage d'un replay en faisant défiler la barre de difficulté en bas de l'écran !
+- La prise en charge du multithreading signifie que même avec peu de FPS, vos clics et vos jugements seront précis !
```
Comment (Pisapou) [inline wiki/Client/Menu_tips/fr.md:108]:
```suggestion
- La prise en charge du multithreading signifie que même avec peu de "FPS", vos clics et vos jugements seront précis !
```

**Review by Ganondorf-1 — DISMISSED** [review]

File: wiki/Client/Menu_tips/fr.md [hunk 16]
```diff
@@ -0,0 +1,121 @@
...
+- Une beatmap peut posséder son propre skin intégré, il suffit juste de copier les éléments du skin dans le dossier de la beatmap. Gardez la même convention de nommage que pour les skins ordinaires.
+- Saviez-vous qu'en copiant des notes dans l'éditeur (`Ctrl` + `C` ou menu `Edit`), le presse-papier contiendra leur position temporelle ainsi que le numéro du combo ? C'est très utile pour rédiger des posts dans le forum et faire des mods !
+- Utilisez le live mapping (`Ctrl` + `Tab`) pour placer très rapidement des notes avec les touches que vous utilisez habituellement pour jouer à osu!mania et Taiko.
+- Faites attention à ne pas superposer un slider avec un autre ou avec lui-même pour satisfaire aux critères de ranking.
+
+## osu!(lazer)
+
+<!-- Source: https://github.com/ppy/osu/blob/master/osu.Game/Screens/Menu/Disclaimer.cs -->
+
+Ces astuces apparaissent dans la [version lazer du client](https://github.com/ppy/osu-lazer), au démarrage, avant la séquence d'introduction.
```
Comment (Walavouchey) [inline wiki/Client/Menu_tips/fr.md:98]: as mentioned in the top comment, add a sentence here that says these tips aren't translated in-game

Comment (Pisapou) [inline wiki/Client/Menu_tips/fr.md:98]:
```suggestion
Ces astuces apparaissent dans la [version lazer du client](https://github.com/ppy/osu-lazer), au démarrage, avant la séquence d'introduction. Notez qu'elles ne sont cependant pas encore traduites dans le jeu, donc uniquement disponibles en anglais.
```

Comment (Pisapou) [reply wiki/Client/Menu_tips/fr.md]: > as mentioned in the top comment, add a sentence here that says these tips aren't translated in-game

Done, i'll just ask a review of ganondorf

**Review by Ganondorf-1 — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**